content/docs/guides/webassembly/resources.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Additional Resources"
+weight: 4
+description: |
+  Additional Resources for using TinyGo with WASM/WASI
+---
+
+Here are a few resources for helping learn more about using TinyGo with WebAssembly.
+
+**Wasm By Example**
+https://wasmbyexample.dev/
+
+**Are We Wasm Yet ? - Part 1**
+https://elewis.dev/are-we-wasm-yet-part-1
+
+**Are We Wasm Yet ? - Part 2**
+https://elewis.dev/are-we-wasm-yet-part-2
+
+**Shrink Your TinyGo WebAssembly Modules by 60%**
+https://www.fermyon.com/blog/optimizing-tinygo-wasm
+
+**Writing a WebAssembly Service in TinyGo for Wagi and Spin**
+https://www.fermyon.com/blog/tinygo-webassembly-favicon-server
+
+**wazero - TinyGo**
+https://wazero.io/languages/tinygo/
```

Comment:
This one contains some outdated information on `malloc` and `free`. I'm not sure we should encourage people reading this? (Or perhaps send a message to the blog post writers to update?)

Reply:
> Or perhaps send a message to the blog post writers to update?

That for sure! cc @codefromthecrypt

Reply:
the jury is still out on the malloc/free stuff, but coming in soon. We need to see benchmark before after and it is nearly there.

Reply:
basically there are a lot of people using the existing approach (which was reviewed by several) we owe it when doing an update to say what the new is and what it will cost you https://github.com/tetratelabs/wazero/pull/1390

Reply:
@codefromthecrypt I'm not sure what you mean? `malloc` and `free` has been safe for use since https://github.com/tinygo-org/tinygo/pull/3148 and I doubt we'll ever change that back.
A hand rolled `malloc` will have a much bigger chance of being subtly wrong.

Reply:
@aykevl Thanks. We we were missing the context

What you said "malloc and free has been safe for use since https://github.com/tinygo-org/tinygo/pull/3148 and I doubt we'll ever change that back." should be in the PR desc as rationale to use the CGO as the only way and remove the others, especially as the benchmarks don't show any perf regression. The same PR can update the site docs to only say one way (CGO malloc/free). @lburgazzoli can you update https://github.com/tetratelabs/wazero/pull/1390 accordingly? Once that's in, I'll we can remove the manual exports from tinymem, leaving it only used for slice helper functions.

Reply:
> What you said "malloc and free has been safe for use since [tinygo-org/tinygo#3148](https://github.com/tinygo-org/tinygo/pull/3148) and I doubt we'll ever change that back." should be in the PR desc as rationale to use the CGO as the only way and remove the others, especially as the benchmarks don't show any perf regression.

@codefromthecrypt  I'm not sure I follow. Was this a reply to me, or which PR are you referring to?

Reply:
> @lburgazzoli can you update https://github.com/tetratelabs/wazero/pull/1390 accordingly?

I think @codefromthecrypt was asking @lburgazzoli to update tetratelabs/wazero#1390 accordingly. :smile_cat: